scripts/docs-versions.sh

```diff
@@ -1,10 +1,20 @@
-#! /usr/bin/env bash
+#! /usr/bin/env sh
 
 git tag --list | \
 grep -v -E '0\.1?[0-9]\.' | \
 grep '^[0-9]' | \
-tac | \
-awk '{print "    {\"name\": \"" $1 "\", \"url\": \"/api/" $1 "/\"}," }' | \
-sed -e "1 s/^/{\n  \"versions\": [\n    {\"name\": \"$(cat src/VERSION)\", \"url\": \"\/api\/master\/\", \"released\": false},\n/" \
-    -e "\$ s/.\$//" \
-    -e "\$ a \  ]\n}"
+awk '
+    BEGIN {
+        print "}"
+        print "  ]"
+    }
+    { print "    {\"name\": \"" $1 "\", \"url\": \"/api/" $1 "/\"}," }
+    END {
+        getline < "src/VERSION"
+        print "    {\"name\": \"" $0 "\", \"url\": \"/api/master/\", \"released\": false},"
+        print "  \"versions\": ["
+        print "{"
+    }
+    ' | \
+sed -e "3 s/.\$//" | \
+tac
```

Comment:
This is, quite literally, backwards

```awk
BEGIN {
  print "{"
  print "  \"versions\": ["
  getline current_version < "src/VERSION"
  printf "    {\"name\": \"" current_version "\", \"url\": \"/api/master/\", \"released\": false}"
}

{
  printf ",\n    {\"name\": \"" $1 "\", \"url\": \"/api/" $1 "/\"}"
}

END {
  print "\n  ]"
  print "}"
}
```

this is cleaner

Reply:
Both are just not ideal solutions. Buy yeah, between backwards code and leading commans the latter is probably better.

Reply:
Oh I missed the line break, so this is really pretty good 👍